language/documentation/book/src/integers.md

```diff
@@ -10,7 +10,9 @@ Move supports three unsigned integer types: `u8`, `u64`, and `u128`. Values of t
 
 ## Literals
 
-Literal values for these types are specified either as a sequence of digits (e.g.,`112`) or as hex literals, e.g., `0xFF`. The type of the literal can optionally be added as a suffix, e.g., `112u8`. If the type is not specified, the compiler will try to infer the type from the context where the literal is used. If the type cannot be inferred, it is assumed to be `u64`.
+Literal values for these types are specified either as a sequence of decimal digits with/without underscore separators for grouping (e.g.,`112`, `234_567`, `97_`) or as hex literals, e.g., `0xFF`. The type of the literal can optionally be added as a suffix, e.g., `112u8`, `92_67_u64`. If the type is not specified, the compiler will try to infer the type from the context where the literal is used. If the type cannot be inferred, it is assumed to be `u64`.
```

Comment:
I would suggest leaving this paragraph as-is and explaining the `_` syntactic sugar in the follow-up para.